lib/GTK/Simple/NativeLib.pm6

```diff
@@ -92,18 +92,11 @@ sub gobject-lib is export {
 
 sub find-bundled($lib is copy) {
     # if we can't find one, assume there's a system install
-    my $base = "lib/GTK/$lib";
-    for @*INC {
-        if my @files = ($_.files($base) || $_.files("blib/$base")) {
-            my $files = @files[0]<files>;
-            my $tmp = $files{$base} || $files{"blib/$base"};
+    my $base = "blib/lib/GTK/$lib";
 
-            # copy to a temp dir
-            $tmp.IO.copy($*SPEC.tmpdir ~ '\\' ~ $lib);
+    if my $file = %?RESOURCES{$base} {
+            $file.IO.copy($*SPEC.tmpdir ~ '\\' ~ $lib);
```

Comment:
There seems to be some sort of read/execute protection on the original library path, so leaving out the copy operation breaks execution and throws `Cannot locate native library 'C:\rakudo\share\perl6\site\resources\9E9773D8C824DFA332B1290D3C2BD5BD44ED4890.dll': error
0x7e`, though I am not able to find the origin of this.